docs/04-common/python_requirements.rst

```diff
@@ -19,7 +19,7 @@ For Ubuntu, please run:
 
 .. code-block:: bash
 
-    sudo apt install swig
+    sudo apt install swig4.1
```

Comment:
This package does not exist for Ubuntu 22. Just add a warning that this is required for Ubuntu 24. I mean, the same command with swig4.1 for ubuntu 24.

See an example on how to create this warnings [here](https://dds-recorder.readthedocs.io/en/latest/rst/installation/windows.html)